x/ccv/provider/keeper/keeper.go

```diff
@@ -107,7 +105,6 @@ func (k Keeper) mustValidateFields() {
 	ccv.PanicIfZeroOrNil(k.clientKeeper, "clientKeeper")             // 9
 	ccv.PanicIfZeroOrNil(k.stakingKeeper, "stakingKeeper")           // 10
 	ccv.PanicIfZeroOrNil(k.slashingKeeper, "slashingKeeper")         // 11
-	ccv.PanicIfZeroOrNil(k.evidenceKeeper, "evidenceKeeper")         // 12
 	ccv.PanicIfZeroOrNil(k.distributionKeeper, "distributionKeeper") // 13
```

Comment:
Do we re-number these or is there supposed to be 11 followed by 13 now in the comments?